src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/BlobManager.java

```diff
@@ -204,7 +206,19 @@ private void moveFileToRejectedContainer(
             ? AccessCondition.generateLeaseCondition(leaseId)
             : AccessCondition.generateEmptyCondition();
 
-        inputBlob.deleteIfExists(DeleteSnapshotsOption.NONE, deleteCondition, null, null);
+        try {
+            inputBlob.deleteIfExists(DeleteSnapshotsOption.NONE, deleteCondition, null, null);
+        } catch (StorageException e) {
+            //if lease lost retry
+            log.info("Deleting File {} got error ", fileName, e);
+            if (e.getHttpStatusCode() == HttpURLConnection.HTTP_PRECON_FAILED
+                && StorageErrorCodeStrings.LEASE_LOST.equals(e.getErrorCode())) {
```

Comment:
could it be `e.getErrorCode() == StorageErrorCodeStrings.LEASE_LOST`, similar to above?

Reply:
it is string